### PR TITLE
Bump ProtoData and Validation

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR updates the dependency on ProtoData and Validation.

### CODE REVIEW

File: mc-java-annotation/build.gradle.kts [hunk 1]
```diff
@@ -68,3 +68,11 @@ tasks.test {
 tasks.withType<ProcessResources>().configureEach {
     duplicatesStrategy = DuplicatesStrategy.INCLUDE
 }
+
+modelCompiler {
+    java {
+        codegen {
+            rejections().enabled.set(false)
```
Comment (armiol) [inline mc-java-annotation/build.gradle.kts:75]: Let's also comment on why we disable the plugin here.

My guess in _this_ case is that we disable it _just in case_ there is a conflict.

File: mc-java-base/build.gradle.kts [hunk 2]
```diff
@@ -53,3 +53,11 @@ project.afterEvaluate {
         duplicatesStrategy = DuplicatesStrategy.INCLUDE
     }
 }
+
+modelCompiler {
+    java {
+        codegen {
+            rejections().enabled.set(false)
```
Comment (armiol) [inline mc-java-base/build.gradle.kts:60]: Same as above.

**Review by armiol — CHANGES_REQUESTED** [review]
@alexander-yevsyukov LGTM in general. However, I would like to discuss my comment (it's basically the same repeated in two use cases).

File: mc-java-annotation/build.gradle.kts (hunk 1 above)
Comment (alexander-yevsyukov) [reply mc-java-annotation/build.gradle.kts:75]: I've checked . We use validation constraints in this module. So we can turn off only rejection codegen.

**Review by armiol — APPROVED** [review]
LGTM.

**Pull request merged**